packages/fuels-programs/src/contract/loader.rs

```diff
@@ -236,6 +236,18 @@ impl Contract<Loader<BlobsNotUploaded>> {
             .await
     }
 
+    /// Deploys the loader contract after uploading the code blobs,
+    /// skipping deploy if it was already deployed with this ID.
```

Comment:
Not sure if I'd say deployed with I'd.

It's deployed with this load configuration and that gave it the I'd.

Maybe avoid that altogether and say I'd there is no contract with this I'd or something like that.